packages/fgtclb/academic-jobs/Classes/Property/TypeConverter/ImageUploadConverter.php

```diff
@@ -159,27 +183,26 @@ private function getOrCreateTargetFolder(string $targetFolderIdentifier): Folder
                         'Target upload folder "%s" does not exist and creation of forbidden by TypeConverter configuration',
                         $targetFolderIdentifier
                     ),
-                    1690527439
+                    1753712957
                 );
             }
         }
 
         if (!$uploadFolder instanceof Folder) {
             throw new TypeConverterException(
                 sprintf('Target upload folder "%s" is not accessible', $targetFolderIdentifier),
-                1690527459
+                1753712962
             );
         }
 
         return $uploadFolder;
     }
 
-    /**
-     * @param array{__identity?: string} $uploadedFileInformation
-     */
-    private function handleNoFileUploaded(array $uploadedFileInformation): ?ExtbaseFileReference
+    // @todo Check how the `__identifier` key was generated in TYPO3 version below 12 before restoring the function
```

Comment:
Outcommeted code ? That should be either removed or resolved within the pull-request before beeing merge ready.